tests/functional/conftest.py

```diff
@@ -0,0 +1,11 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+
+@pytest.fixture
+def selenium(selenium):
+    selenium.maximize_window()
```

Comment:
I think we should specify the window size so that tests are consistent regardless of the resolution of the machine running the browser. See [set_window_size](http://seleniumhq.github.io/selenium/docs/api/py/webdriver_remote/selenium.webdriver.remote.webdriver.html#selenium.webdriver.remote.webdriver.WebDriver.set_window_size). I note that the CasperJS tests defaulted to 1000x600. We might want to pick something a little more standard.